docs/source/tutorials/configurable.rst

```diff
@@ -0,0 +1,106 @@
+==============================
+Configuration and plugin setup
+==============================
+
+Maybe you :doc:`played with commands </tutorials/playing-command>` for your
+plugin and now you want to make your plugin configurable. If you run an
+instance of Sopel yourself, you probably had to open and edit its configuration
+file.
+
+Usually located in your home directory, under the ``.sopel/`` folder, the
+configuration file is an INI file with sections defined by Sopel's core and by
+plugins. In this tutorial, let's see how to declare and use a configuration
+section dedicated to your plugin.
+
+
+Declaring your configuration
+============================
+
+To declare a configuration section, you must first create a subclass of
+:class:`~sopel.config.types.StaticSection`, and define attributes::
+
+    from sopel.config import types
+
+    class MyFirstPlugin(types.StaticSection):
```

Comment:
Decision point: Do we reinforce the core convention of naming the class `PluginNameSection` in this tutorial?

Even if we decide to promote a different conventional name format, it's probably wise to guide the user toward some meaningful suffix, like how most (built-in/stdlib) exceptions in Python end with `Error`.

Reply:
Good point! I'll try to use the same plugin name in the tutorial so it's easier to keep track of that. I believe having consistent naming will be enough for the tutorial.